fe/fe-core/src/main/java/org/apache/doris/catalog/ColumnIdFlushDaemon.java

```diff
@@ -0,0 +1,134 @@
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.catalog;
+
+import org.apache.doris.alter.AlterLightSchChangeHelper;
+import org.apache.doris.common.util.MasterDaemon;
+
+import com.google.common.collect.Maps;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.ReadWriteLock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+/**
+ * note(tsy): this class is temporary, make table before 1.2 to enable light schema change
+ */
+public class ColumnIdFlushDaemon extends MasterDaemon {
+
+    /**
+     * db name -> (tbl name -> status)
+     */
+    private final Map<String, Map<String, FlushStatus>> resultCollector;
+
+    private final ReadWriteLock rwLock;
+
+    public ColumnIdFlushDaemon() {
+        super("colum-id-flusher", TimeUnit.HOURS.toMillis(1));
+        resultCollector = Maps.newHashMap();
+        rwLock = new ReentrantReadWriteLock();
+    }
+
+    @Override
+    protected void runAfterCatalogReady() {
+        flush();
+    }
+
+    private void flush() {
+        List<Database> dbs = Env.getCurrentEnv().getInternalCatalog().getDbs();
+        rwLock.writeLock().lock();
+        try {
+            for (Database db : dbs) {
+                db.getTables()
+                        .stream()
+                        .filter(table -> table instanceof OlapTable)
+                        .map(table -> (OlapTable) table)
+                        .filter(olapTable -> !olapTable.getTableProperty().getUseSchemaLightChange())
+                        .forEach(table -> {
+                            try {
+                                table.writeLock();
+                                if (table.getTableProperty().getUseSchemaLightChange()) {
+                                    table.writeUnlock();
+                                    return;
+                                }
+                                new AlterLightSchChangeHelper(db, table).enableLightSchemaChange();
+                                table.writeUnlock();
+                                recordResult(db.getFullName(), table.getName(), FlushStatus.ok());
+                            } catch (IllegalStateException e) {
+                                recordResult(db.getFullName(), table.getName(), FlushStatus.failed(e.getMessage()));
+                            }
+                        });
+            }
+        } finally {
+            rwLock.writeLock().unlock();
+        }
+    }
+
+    private void recordResult(String dbName, String tableName, FlushStatus status) {
+        resultCollector.putIfAbsent(dbName, Maps.newHashMap());
+        Map<String, FlushStatus> tableToStatus = resultCollector.get(dbName);
+        tableToStatus.put(tableName, status);
+    }
+
+    public Map<String, Map<String, FlushStatus>> getResultCollector() {
+        return resultCollector;
```

Comment:
If use does not set enable_convert_light_weight_schema_change to true, the the result collector is empty. User will not know if the flush process is stopped or it is not begin.

I think the show xxx comand should return the tables that not convert to light weigth schema change, it has no relation with the config "enable_convert_light_weight_schema_change":
1. if all tables are converted, then the command return empty.
2. it should return (1) the tables that are not converted  (2) the tables that have been converted but meet errors and the error message.

Reply:
OK, I'll mod it.